postprocessing/pp-h264.c

```diff
@@ -78,7 +76,11 @@ int janus_pp_h264_create(char *destination) {
 	vStream->time_base = (AVRational){1, 90000};
 	vStream->codec->width = max_width;
 	vStream->codec->height = max_height;
+#if LIBAVCODEC_VER_AT_LEAST(53, 0)
+	vStream->codec->pix_fmt = AV_PIX_FMT_YUV420P;
+#else
```

Comment:
Yep, that's why I was thinking that some of the smaller generic fixes we might merge right away, either as a smaller PR or by applying them directly to the code, without waiting for this larger PR to be merged. This particular PR will need some more time, especially considering that as anticipated we have a couple of other PRs still waiting to be merged that may need a bit of refactoring in your effort too.